pkg/rc/replication_controller.go

```diff
@@ -893,12 +903,11 @@ func (rc *replicationController) watchHealth() bool {
 
 	for {
 		select {
-		case err, ok := <-errCh:
-			// TODO rewrite once we understand why nil errors were sent to this chan
+		case err := <-errCh:
 			if err != nil {
 				rc.logger.WithError(err).Errorln("Node transfer health checker sent error")
 			} else {
-				rc.logger.Errorln("Node transfer health checker sent nil error. Ok was: %v", ok)
+				rc.logger.Errorln("Node transfer health checker sent nil error")
```

Comment:
just curious, did we see this happen in our testing?

Reply:
I didn't see it, although there is of course a chance I missed it